pkg/crossplanebroker/crossplanebroker.go

```diff
@@ -211,14 +212,17 @@ func (b *CrossplaneBroker) LastOperation(ctx context.Context, instanceID string,
 		if err != nil {
 			return domain.LastOperation{}, crossplane.ConvertError(ctx, err)
 		}
-		logger.Info("finish-provision")
 		if err := sb.FinishProvision(ctx); err != nil {
 			return domain.LastOperation{}, crossplane.ConvertError(ctx, err)
 		}
+		logger.WithData(lager.Data{"reason": condition.Reason, "message": condition.Message}).Info("provision-succeeded")
 	case v1alpha1.ReasonCreating:
 		op.State = domain.InProgress
-	default:
+		logger.WithData(lager.Data{"reason": condition.Reason, "message": condition.Message}).Info("provision-in-progress")
+	case v1alpha1.ReasonUnavailable:
+	case v1alpha1.ReasonDeleting:
```

Comment:
```suggestion
	case v1alpha1.ReasonUnavailable, v1alpha1.ReasonDeleting:
```

otherwise you need [a `fallthrough` statement](https://github.com/golang/go/wiki/Switch#fall-through)

Reply:
That's what I get for merging before the review 🙈

Reply:
Fixed in https://github.com/vshn/crossplane-service-broker-poc/pull/22